selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -529,18 +530,12 @@ fetchActualResults() {
         ACTUAL_RESULTS_URL=${BASE_ACTUAL_RESULTS_URL}${job}"/"
     fi
 
-    # from 'Failed Tests:' to 'Skipped Tests:'
+    # get list of failed tests from CI server
     local actualResults=($(curl -s ${ACTUAL_RESULTS_URL} | \
                            tr '>' '\n' | tr '<' '\n' | tr '"' '\n'  | \
-                           grep -A9999999 "Failed Tests:" | grep -B9999999 "Skipped Tests:" | \
```

Comment:
Do they include skipped tests in Test result? Otherwise we lost something.

Reply:
I am afraid, but url that is used to request failed tests `https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/` doesn't include all failed tests. The correct  url is `https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/#showFailuresLink`
Does this PR cover this usecase?

Reply:
Did you try to curl both urls? :-)

Reply:
Result will be the same.

Reply:
Also there is no section "Skipped tests" in the tests result page https://ci.codenvycorp.com/view/qa/job/che-integration-tests/52/. We doesn't have such statistics.

Reply:
BTW: Jenkins place skipped tests in the list of failed tests. For example, skipped tests:
``` 
org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugStaticInnerClass
org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugLambdaExpressions
org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugAnonymousClass
```
![screenshot from 2017-09-29 14 53 51](https://user-images.githubusercontent.com/1197777/31014948-2fd6f8e2-a526-11e7-8063-e60c5b71975b.png)

had been placed in the list: https://ci.codenvycorp.com/view/qa/job/che-integration-tests/53/
![screenshot from 2017-09-29 14 56 46](https://user-images.githubusercontent.com/1197777/31015007-7b3449ca-a526-11e7-904c-9831ebcc32d4.png)